### ISSUE TITLE
loss convergence 

### ISSUE THREAD

**Issue by legendhua** [assassint2017/MICCAI-LITS2017#1]
Hi, I arranged my equipment environment follow your code. But the train loss is unchanged, then I add val loss and find the val loss is also keeping unchanged. I changed the learning rate bigger, but still not effect. Have you encountered this problem?

**Comment by assassint2017** [comment]
In my experiments, at the beginning of the training process, the loss value dropped very quickly and quickly reached about 0.1.

**Comment by assassint2017** [comment]
Which loss function did you use? Focal loss or Dice loss, i use Dice loss in my experiments

**Comment by legendhua** [comment]
Focal loss is my choose, I will try the dice loss, and did you apply the train process with focal loss. Thanks for you advice.

**Comment by assassint2017** [comment]
I haven't had time to try focal loss. The specific implementation of focal loss is quite variable. Hyperparameters also affect performance, and the coefficients that normalize the loss function are also worthy of attention. In my code, I use the same strategy as the original paper. But for different problems, in order to get good performance you may need to try other different strategies. In short, using focal loss and achieving good results is not an easy task, and that is  why I prefer Dice loss.

**Comment by legendhua** [comment]
Thanks for you detailed reply, 非常感谢！！

**Comment by assassint2017** [comment]
~=o(*^▽^*)o~♪

**Comment by John1231983** [comment]
@assassint2017 : What is your rank in leaderboard? Thanks
